<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>384</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QtPaint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/app.png</normaloff>:/icons/app.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblImageSize">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Image size</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="PaintArea" name="paintArea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lblWarning">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Warning: the image will be downscaled to fit into the size limit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_image"/>
   <addaction name="actionOpen_file"/>
   <addaction name="actionSave_file"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionRotate_left"/>
   <addaction name="actionRotate_right"/>
   <addaction name="actionFlip_horizontal"/>
   <addaction name="actionFlip_vertical"/>
   <addaction name="actionSelect_color"/>
   <addaction name="separator"/>
   <addaction name="actionLayers"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSend"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionOpen_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="toolTip">
    <string>Open image file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save file</string>
   </property>
   <property name="toolTip">
    <string>Save image file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSelect_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/color-picker.png</normaloff>:/icons/color-picker.png</iconset>
   </property>
   <property name="text">
    <string>Pen color</string>
   </property>
   <property name="toolTip">
    <string>Open color selection dialog</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionNew_image">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New image</string>
   </property>
   <property name="toolTip">
    <string>Create blank image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste image from clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy image to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionFlip_horizontal">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/flip-horizontal.png</normaloff>:/icons/flip-horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Flip horizontal</string>
   </property>
   <property name="toolTip">
    <string>Flips image on horizontal axis</string>
   </property>
  </action>
  <action name="actionFlip_vertical">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/flip-vertical.png</normaloff>:/icons/flip-vertical.png</iconset>
   </property>
   <property name="text">
    <string>Flip vertical</string>
   </property>
   <property name="toolTip">
    <string>Flips image on vertical axis</string>
   </property>
  </action>
  <action name="actionRotate_left">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rotate-left.png</normaloff>:/icons/rotate-left.png</iconset>
   </property>
   <property name="text">
    <string>Rotate left</string>
   </property>
   <property name="toolTip">
    <string>Rotates image counter-clockwise</string>
   </property>
  </action>
  <action name="actionRotate_right">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rotate-right.png</normaloff>:/icons/rotate-right.png</iconset>
   </property>
   <property name="text">
    <string>Rotate right</string>
   </property>
   <property name="toolTip">
    <string>Rotates image clockwise</string>
   </property>
  </action>
  <action name="actionLayers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/layers.png</normaloff>:/icons/layers.png</iconset>
   </property>
   <property name="text">
    <string>Layers</string>
   </property>
   <property name="toolTip">
    <string>Show layers menu</string>
   </property>
  </action>
  <action name="actionSend">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/send.png</normaloff>:/icons/send.png</iconset>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/clear-2.png</normaloff>:/icons/clear-2.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintArea</class>
   <extends>QWidget</extends>
   <header>paintarea.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
